.gitignore

```diff
@@ -32,6 +32,8 @@ yarn-error.log
 /features/discovery/activemq-data
 /features/topology-map/org.opennms.features.topology.app/www-test/
 /opennms-alarms/jms-northbounder/activemq-data/
+ui/node_modules/
+ui/package-lock.json
```

Comment:
`package-lock.json` should generally be checked into git, so that upstream changes don't break builds

Reply:
The build uses yarn and produces a yarn.lock file, which is checked in.
This was more of a safety from some devs who were using NPM locally and creating/checking in the unneeded package-lock.

Reply:
I remove the line though, if we're not worried about that

Reply:
I think it's safe to ignore given we use and check-in the `yarn.lock`.

@RangerRick do you think we should check-in both?

Reply:
Ah, sorry, if you're using yarn then never mind, I missed that.